solidity/contracts/examples/SpecAndRunRequester.sol

```diff
@@ -25,10 +25,19 @@ contract SpecAndRunRequester is Chainlinked, Ownable {
     tasks[3] = "ethtx";
 
     ChainlinkLib.Spec memory spec = newSpec(tasks, this, "fulfill(bytes32,bytes32)");
+
+    // httpget params
     spec.add("url", "https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD,EUR,JPY");
+
+    // jsonparse params
     string[] memory path = new string[](1);
     path[0] = _currency;
     spec.addStringArray("path", path);
+
+    // ethtx params (callback for the CL node is actually the oracle)
+    spec.add("address", toAsciiString(oracle));
+    spec.add("functionSelector", fromCode(bytes4(keccak256("fulfill(uint256,bytes32)"))));
```

Comment:
Technically no.

* fulfill(bytes32,bytes32) is the call back for the `Consumer`: https://github.com/smartcontractkit/chainlink/blob/master/solidity/contracts/examples/Consumer.sol#L35
* fulfill(uint256,bytes32) is the callback for the `Oracle`: https://github.com/smartcontractkit/chainlink/blob/master/solidity/contracts/Oracle.sol#L99-L102

These parameters are for `EthTx`, which is how the CL node interacts with the CL Oracle. This is now a non-issue though because we've opted to go a different route exactly because of the confusion this causes.